siliconcompiler/tools/yosys/syn_asic.py

```diff
@@ -226,9 +226,12 @@ def get_synthesis_libraries(lib):
             # Mark dont use
             for lib_file in get_synthesis_libraries(lib):
                 lib_content.append(
-                    markDontUse.processLibertyFile(lib_file, dont_use,
-                                                   chip.get('option', 'quiet',
-                                                            step=step, index=index)))
+                    markDontUse.processLibertyFile(
```

Comment:
@gadfort 
Just saying you could you reuse it here and only have one ternary expression.